conf/st2.conf.sample

```diff
@@ -189,6 +189,8 @@ redirect_stderr = False
 [messaging]
 # URL of all the nodes in a messaging service cluster.
 cluster_urls =  # comma separated list allowed here.
+# Compression algorithm to use for compressing the payloads which are sent over the message bus. Valid values include: zstd, lzma, bz2, gzip. Defaults to no compression.
```

Comment:
do we need extra libraries when using the various compression algorithms?

Reply:
Only for zstandard, but we bundle that with the StackStorm virtual environment inside the deb and rpm packages.